src/langcheck/metrics/en/pairwise_text_quality.py

```diff
@@ -21,6 +100,9 @@ def pairwise_comparison(
     sources_b: Optional[List[str] | str] = None,
     reference_outputs: Optional[List[str] | str] = None,
     enforce_consistency: bool = True,
+    calculated_confidence: bool = False,
+    k: int = 5,
+    n: int = 5,
```

Comment:
Let's also add a seed parameter so that users can make the few shot prompts deterministic if they want to